apps/prairielearn/src/lib/db-types.ts

```diff
@@ -507,18 +507,21 @@ export const InstanceQuestionSchema = z.object({
   id: IdSchema,
   incremental_submission_points_array: z.array(z.number()).nullable(),
   incremental_submission_score_array: z.array(z.number()).nullable(),
+  instructor_question_number: z.string().nullable(),
   last_grader: IdSchema.nullable(),
   last_submission_score: z.number().nullable(),
   manual_points: z.number().nullable(),
   max_submission_score: z.number().nullable(),
-  modified_at: DateFromISOString,
+  modified_at: z.string(),
   number: z.number().nullable(),
   number_attempts: z.number(),
   open: z.boolean(),
   order_by: z.number().nullable(),
   points: z.number().nullable(),
   points_list: z.array(z.number()).nullable(),
   points_list_original: z.array(z.number()).nullable(),
+  qid: z.string().nullable(),
+  question_number: z.string().nullable(),
```

Comment:
Note that these are based on the query in `instructorAssessmentInstance`, and should not be modified here because of that different query. You can, instead, have a new schema in the page router code that extends this one, like:

```ts
const InstanceQuestionExtendedSchema = InstanceQuestionSchema.extend({
  modified_at: z.string(),
  qid: z.string().nullable(),
  // ...
});

Reply:
Thanks! I have added a new schema in the router and used your code above to extend the `InstanceQuestionSchema` in commit 9c3e8cd.